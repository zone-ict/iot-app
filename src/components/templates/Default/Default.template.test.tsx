import React from 'react';
import renderer from 'react-test-renderer';
import DefaultTemplate from './Default.template';

describe('Default Template', () => {
  it('should render', () => {
    const tree = renderer.create(<DefaultTemplate>Test</DefaultTemplate>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
