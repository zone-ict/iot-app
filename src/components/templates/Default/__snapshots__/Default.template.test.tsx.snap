// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default Template should render 1`] = `
<main>
  Test
</main>
`;
